docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,96 @@ keywords:
-This ensures that your Workflows can be resumed automatically in the presence of failure, without the extensive boilerplate code and complexity that comes with using raw EDA frameworks directly.
+- **Smaller failure domains** \- When teams operate in the same monolithic Namespace, everything is available to everyone, and mis-behaving workers can trigger rate limits that affect all teams operating in that monolithic Namespace.
+  Nexus enables each team to have their own Namespace for improved security, troubleshooting, and fault isolation.
 
-With Temporal Nexus, inter-service communication and durable execution can span teams, Namespaces, and regions all from within the Temporal context.
-Nexus preserves a familiar API programming model that supports both synchronous and asynchronous patterns, is suitable for low-latency and long-running operations, and uses an optimized form of the Temporal queue-based Worker architecture to preserve the benefits of EDA at runtime.
+- **Multi-region use cases** \-
+Nexus requests in Temporal Cloud are routed across a global mTLS-secured Envoy mesh.
+Reliable execution is provided with built-in Nexus Machinery and Temporal's queue-based Worker architecture.
 
-Temporal Nexus provides an integrated Temporal SDK experience, built-in Nexus Machinery, first-class observability, and enables each team to have their own Namespace for improved security, troubleshooting, and blast radius isolation.
-It allows multiple teams to collaborate autonomously while maintaining a clean separation of concerns, ensuring that applications are scalable, secure, and easy to manage as they grow.
+**Advantages of Temporal Nexus:**
 
-With Temporal Nexus, teams are able to:
+- Reliably connect Temporal Applications within and across Namespaces.
+- Service contracts suitable for sharing across teams.
+- Abstract underlying Temporal primitives, like Workflows – or execute arbitrary code.
+- Modular design that streamlines multi-team development.
+- Improved security and blast-radius isolation with isolated Namespaces.
+- Same queue-based Worker architecture, no bespoke service deployments needed.
 
-- **Easily scale and manage applications** by breaking them into modular components developed separately while maintaining smooth integration through clear service contracts.
-- **Run Workflows across teams and Namespaces**, enabling durable execution and custom code to operate seamlessly across different parts of the system.
-- **Enhance security by isolating different parts of the application**, reducing the risk and impact of potential issues.
-- **Continue using familiar tools like queue-based Workers**, without the need for complex new setups or deployments.
+<RelatedReadContainer>
+  <RelatedReadItem path="https://youtu.be/qqc2vsv1mrU?feature=shared&t=2082" text="Nexus keynote & demo - Replay video" archetype="replay-talk" />
```

Comment:
Might have to look into getting a Nexus icon here.

Reply:
that would be awesome!